.travis.yml

```diff
@@ -1,11 +1,17 @@
 language: php
 
 php:
+  - 7.1
   - 7.2
 
 install:
   - travis_retry composer self-update && composer --version
   - travis_retry composer update --no-interaction
 
 script:
-  - composer all
```

Comment:
why not to encapsulate those commands into `all` script?

Reply:
I need phpunit with specific settings for UI (to generate clover). But I may do something like "all-ci" or so.